lambda_backend/route_microservice/src/common/cognito.ts

```diff
@@ -36,5 +34,14 @@ export const adminGetCognitoUserDetails = async (Username: string): Promise<Cogn
 const parseUserAttributes = (UserAttributes: AttributeType[]): CognitoUserDetails => {
   const userEmail = UserAttributes.filter((attribute) => attribute.Name === 'email')[0]
     .Value as string;
-  return { userEmail };
+  const userRole = UserAttributes.filter((attribute) => attribute.Name === 'custom:role')[0]
+    .Value as UserRole;
+  return { userEmail, userRole };
 };
+
+interface CognitoUserDetails {
+  userEmail: string;
+  userRole: UserRole;
+}
+
+type UserRole = 'admin' | 'user';
```

Comment:
Extract out into some common types file or something?

Reply:
i feel like for now its only related to this module. the rest not using it much yet

Reply:
ok i ll extracterino

Reply:
There's another file with this tho

Reply:
different microservices